Automattic/jetpack
# Social: Poll for share status after reshare

## Proposed changes:

* Poll for share status after reshare

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to post editor
* Create post and don't share the post to any network either by disabling "Share this post" or disabling all connections.
* Keep Network tab open
* Now reshare the post
* Confirm that the polling for share status starts and finishes when there is new data
* Confirm that you can now see "View sharing history" button
* Click on it to see the shares
* Now reshare again
* Confirm that the polling starts and fetches the new posts.
* To test the time out, reshare only to Instagram without the post having an image or SIG.
* Confirm that the request times out after one minute after resharing only to Instagram without image.
* Turn OFF the feature flag, confirm that the polling doesn't start after reshare
* Confirm that the polling doesn't start on WPCOM sites because they don't have the feature enabled.
